radio/src/gui/gui_common.cpp

```diff
@@ -1101,6 +1101,18 @@ bool isTrainerModeAvailable(int mode)
 #endif
   }
 
+  if (mode == TRAINER_MODE_CRSF) {
+
+#if !defined(CROSSFIRE)
+    return false;
+#else
+    if ((!IS_INTERNAL_MODULE_ENABLED() && !IS_EXTERNAL_MODULE_ENABLED()) ||
+         (!isModuleCrossfire(INTERNAL_MODULE) &&
+          !isModuleCrossfire(EXTERNAL_MODULE)))
+      return false;
```

Comment:
Looks like this earmarked for ELRS v4? Might be worth checking if `CRSF_ELRS_MIN_VER(module, 4, 0)` can be added to the mix here...

Reply:
Will do.

Reply:
Fixed and I also found a bug in the ELRS version check macro, which I fixed!